tests/t022_filter_kernel.py

```diff
@@ -42,7 +42,7 @@ def fixup(self, cflags, result):
         # Linux v4.17 (x86_64) changed syscall routines
         major, minor, release = uname[2].split('.')
         if uname[0] == 'Linux' and uname[4] == 'x86_64' and \
-           int(major) >= 4 and int(minor) >= 17:
+           ((int(major) >= 4 and int(minor) >= 17) or int(major) >= 5):
```

Comment:
I think it should be `(int(major) == 4 and ...)`. Also as later distro will ship later versions so I think it'd better checking major version being more than 5 first, like:

```python
int(major) >= 5 or (int(major) == 4 and int(minor) >= 17):
```

Reply:
I think so too. Should i change my codes to forced pull-request?

Reply:
Yes, you can just amend the commit and force-push to the same branch.

Reply:
I changed my codes. is that done?

Reply:
Yes, I can see your change now.  But it's better letting me know you updated the branch by leaving a comment like you did.

Reply:
Aha, i got it. Thank you for advising me.